VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -29,18 +29,34 @@ import Foundation
 class VideoSettings: NSObject
 {
     var title: String?
+    {
+        didSet
+        {
+            self.title = trim(self.title)
+        }
+    }
+    
     var desc: String?
+    {
+        didSet
+        {
+            self.desc = trim(self.desc)
+        }
+    }
+    
     var privacy: String?
     var users: [String]? // List of uris of users who can view this video
     var password: String?
 
     init(title: String?, description: String?, privacy: String?, users: [String]?, password: String?)
     {
-        self.title = title
-        self.desc = description
+        super.init()
+        
         self.privacy = privacy
         self.users = users
         self.password = password
+        self.title = self.trim(title)
+        self.desc = self.trim(description)
```

Comment:
This seems redundant. We're trimming here before we invoke the setter. And then above in the setting we're trimming as well. Should probably not trim here in init and just let the setter handle it all.